packages/frontend/core/package.json

```diff
@@ -57,6 +57,7 @@
     "cmdk": "^1.0.0",
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 50]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->remove this ?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7459?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
I found the `cmdk` is used in the core package. Can it be safely removed?

https://github.com/toeverything/AFFiNE/blob/40e381e2723fdedc626f6c4c27c44d4eceeee496/packages/frontend/core/src/modules/quicksearch/views/cmdk.tsx#L4

Reply:
cleanup work should be done in separated pr